src/components/CVE-2021-41773/CVE-2021-41773.tsx

```diff
@@ -23,20 +23,39 @@ const description_userguide =
     "       Eg: 2.4.49\n\n" +
     "Step 5: Click Exploit to commence the RCE’s operation.\n\n" +
     "Step 6: View the Output block below to view the results of the attack vectors execution.";
+const sourceLink = "cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-41773"; // Link to the source code (or Kali Tools).
+const tutorial = ""; // Link to the official documentation/tutorial.
+const dependencies = ["CVE 2021-41773"]; // Contains the dependencies required by the component.
```

Comment:
I do not think CVE 2021-41773 is the dependency. It should be python3. 
```suggestion
const dependencies = ["python3"]; // Contains the dependencies required by the component.
```